docs/getting-started-guides/ubernetes-lite.md

```diff
@@ -0,0 +1,190 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+Create the first cluster, but pass MULTIZONE to tell the cluster to manage multiple zones; creating nodes in us-central1-a.
+
+```
+MULTIZONE=1 KUBE_GCE_ZONE=us-central1-a cluster/kube-up.sh
+```
+
+View the nodes; you can see that they are tagged with the zone they are in (they are all in us-central1-a so far):
+
+```
+kubectl get nodes
+```
+
+Add more nodes to the existing cluster, reusing the existing master, running in a different zone (us-central1-b):
+
+```
+KUBE_USE_EXISTING_MASTER=true MULTIZONE=1 KUBE_GCE_ZONE=us-central1-b cluster/kube-up.sh
+```
+
+View the nodes again; 3 more nodes should have launched and be tagged in us-central1-b:
+
+```
+kubectl get nodes
+```
```

Comment:
Suggest adding the following here to address @davidopp 's comment about highly available master components.  Then we should merge this, and flesh it out further in a separate PR (that I can work on if you're still busy with other things @justinsb ):

```
Note that the default kube-up.sh script currently still creates all master components in a single availability zone, so if that particular zone fails, the master components will experience an outage until that zone is restored.  If you require a highly available set of master components, please follow the documented steps to achieving [high ava](https://github.com/kubernetes/kubernetes/blob/master/docs/admin/high-availability.md) But the pods in other zones will continue to run, and service load-balancers will balance traffic across those remaining pods.  If one of the zones not hosting